python/paddle/incubate/optimizer/modelaverage.py

```diff
@@ -169,14 +181,22 @@ class ModelAverage(Optimizer):
 
     """
 
+    helper: LayerHelperBase | None
```

Comment:
```suggestion
    helper: LayerHelper
```

没有必要用 `LayerHelperBase` 吧？